README.md

```diff
@@ -78,6 +78,28 @@ Contains:
 
 4. Rename a new bookmark with right-click
 
+## FAQ
+
+### How can I search for games in subscriptions and owned games at the same time?
```

Comment:
Ah, so you mean a general UI trick in Galaxy. I didn't understand firstly.

I'm not convinced to put info like that in this repo, it fits better in awesome-gog-galaxy or a separate FAQ gist to aim bigger audience.

But here we can be more specific, like:

`Can I group my Humble games and Choice games altogether?`

and then the same solution but slightly modified (with additional filter for humble).

Reply:
Ok i think i cannot edit my own PR, so I will do another one, you can reject this!

Reply:
can't you? And if you edit it on your repo?

Reply:
I think I made it :) have a look!